lxc/config/remote.go

```diff
@@ -27,6 +27,7 @@ type Remote struct {
 	Project  string `yaml:"project,omitempty"`
 	Protocol string `yaml:"protocol,omitempty"`
 	Public   bool   `yaml:"public"`
+	Global   bool   `yaml:"gloabl,omitempty"`
```

Comment:
This shouldn't ever be stored in config so should be `yaml:"-"` to keep it as an internal flag like Static